sdk/loadtestservice/load-testing-rest/samples-dev/fileUpload.ts

```diff
@@ -44,7 +44,7 @@ async function main() {
     throw new Error("Test ID returned as undefined.");
 
   // Uploading .jmx file to a test
-  const fileUploadResult = await client
+  /*const fileUploadResult = await client
```

Comment:
I mentioned earlier we would like to keep this call outside the helper function so please uncomment this code.